src/components/Store.react.js

```diff
@@ -1,46 +1,7 @@
-import {any, includes, isNil, type} from 'ramda';
+import {equals, isNil} from 'ramda';
 import React from 'react';
 import PropTypes from 'prop-types';
 
-/**
- * Deep equality check to know if the data has changed.
- *
- * @param {*} newData - New data to compare
- * @param {*} oldData - The old data to compare
- * @returns {boolean} The data has changed.
- */
-function dataChanged(newData, oldData) {
```

Comment:
Any reason to maintain a custom deep equality test here, instead of just `Ramda.equals`? The tests (particularly [`stda001`](https://github.com/plotly/dash-core-components/blob/3bcc2384576bfc7e9ae12a166f1e38615d9a2c8d/tests/integration/store/test_store_data.py#L13) say they're equivalent.

Reply:
🔪Looks legit.